### DISCUSSION TITLE
Multiple classes inside a breakpoint selector

### DISCUSSION THREAD

**Discussion by nistadev** [tailwindlabs/tailwindcss#11472] [Ideas]
### Problem
Thinking of ideas on how to improve the readability of particular cases when you need to use different styles for multiple properties on a certain breakpoint like in this case below:

```html
<div className="desktop:px-[64px] desktop:py-[24px] desktop:grid-cols-3 desktop:grid-rows-1" />
```

### Possible solution
I think would be great to have something like:
```html
<div className="desktop:(px-[64px] py-[24px] grid-cols-3 grid-rows-1)" />
```

Any thoughts on this?  Not sure if this approach or something similar was already suggested.

**Comment by wongjn** [comment]
There have been many suggestions about adding such a feature. [Here's one such response from a maintainer](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

[Also another thread here](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611) with more responses from maintainers.
